src/controllers/user.controller.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable require-jsdoc */
+import db from '../database/models';
+
+/**
+ * @exports UserController
+ * @class UserController
+ * */
+
+class UserController {
+  /**
+   * Get users and their corresponding profiles
+   * @async
+   * @static
+   * @param {object} req - Request object
+   * @param {object} res - Response object
+   * @param {object} next - The next middleware
+   * @returns {json} Returns json object
+   * @memberof UserController
+   */
+  static async listUsers(req, res, next) {
+    try {
+      const users = await db.User.findAll({
+        attributes: ['id', 'username', 'email', 'bio', 'avatar'],
+        where: {
+          status: 'active'
+        }
+      });
+      return res.status(200).json({
+        status: 200,
```

Comment:
please remove the status key and the message

Reply:
All right, thank you. I will remove them.